{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2017-2591",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www.securityfocus.com/bid/95670",
          "name": "95670",
          "refsource": "BID"
        },
        {
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-2591",
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-2591",
          "refsource": "CONFIRM"
        },
        {
          "url": "https://pagure.io/389-ds-base/issue/48986",
          "name": "https://pagure.io/389-ds-base/issue/48986",
          "refsource": "CONFIRM"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "389-ds-base before version 1.3.6 is vulnerable to an improperly NULL terminated array in the uniqueness_entry_to_config() function in the \"attribute uniqueness\" plugin of 389 Directory Server. An authenticated, or possibly unauthenticated, attacker could use this flaw to force an out-of-bound heap memory read, possibly triggering a crash of the LDAP service."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-04-30T12:29Z",
  "lastModifiedDate": "2018-05-02T01:29Z"
}